pkg/parser/yamlparser.go

```diff
@@ -208,20 +219,56 @@ func (doc *YamlDocument) IsBuiltIn(commandName string) bool {
 	return utils.FindInArray(builtInCommands, commandName) != -1
 }
 
-func (doc *YamlDocument) IsOrb(commandName string) bool {
+func (doc *YamlDocument) IsOrbReference(commandName string) bool {
 	splittedCommand := strings.Split(commandName, "/")
 
 	if len(splittedCommand) != 2 {
 		return false
 	}
 
-	_, ok := doc.Orbs[splittedCommand[0]]
+	orbName := splittedCommand[0]
+	_, ok := doc.Orbs[orbName]
+
+	return ok
+}
+
+func (doc *YamlDocument) IsOrbCommand(orbCommand string, cache *utils.Cache) bool {
+	splittedCommand := strings.Split(orbCommand, "/")
+
+	if len(splittedCommand) != 2 {
+		return false
+	}
+
+	orbName := splittedCommand[0]
+	commandName := splittedCommand[1]
+
+	orb, ok := doc.Orbs[orbName]
+
+	if !ok {
+		return false
+	}
+
+	var orbInfo *ast.OrbInfo
+
+	orbId := orb.Url.GetOrbID()
+
+	if orb.Url.IsLocal {
+		orbInfo = doc.LocalOrbInfo[orbId]
+	} else {
+		orbInfo = cache.OrbCache.GetOrb(orbId)
+	}
```

Comment:
instead of doing this, we can call `doc.GetOrbInfo`, no ?

Reply:
👍